src/main/java/baza/trainee/service/EventService.java

```diff
@@ -22,10 +22,12 @@ public interface EventService {
 
     /**
      * Create a new event based on the provided EventPublicationDto.
+     *
      * @param newEvent The EventPublicationDto containing information about the event to be created.
+     * @param id
```

Comment:
Please add a description of what 'id' is.

Reply:
Add - user id(now session id, but decided that it would be used id)